src/gen/conditions.rs

```diff
@@ -1577,7 +1611,7 @@ fn test_agg_sig_unsafe_invalid_pubkey() {
 fn test_agg_sig_unsafe_invalid_msg() {
     // AGG_SIG_ME
     assert_eq!(
-        cond_test("((({h1} ({h2} (123 (((49 ({pubkey} ({longmsg} )))))")
+        cond_test("((({h1} ({h2} (123 (((50 ({pubkey} ({longmsg} )))))")
```

Comment:
this was a copy-paste error from before. `AGG_SIG_UNSAFE` is 49 and `AGG_SIG_ME` is 50